pkg/image/importer/client.go

```diff
@@ -1,17 +1,21 @@
 package importer
 
 import (
+	"crypto/tls"
 	"encoding/json"
 	"fmt"
 	"net"
 	"net/http"
 	"net/url"
 	"path"
+	"path/filepath"
 	"time"
 
 	"github.com/golang/glog"
 	gocontext "golang.org/x/net/context"
 
+	"github.com/docker/docker/registry"
```

Comment:
How many dependencies does this bring in?

Reply:
It depends on 55 packages, but it seems almost all of them are already vendored. It brings in 6 additional packages:
```
github.com/docker/distribution/digestset
github.com/docker/docker/api/types/swarm/runtime
github.com/docker/docker/pkg/stringid
github.com/docker/docker/pkg/tarsum
github.com/docker/docker/registry
github.com/docker/docker/registry/resumable
```

Reply:
Ok, I wish they'd factor that package better.  In he most recent moby code, is this in the same place and is it better/worse/same for dependencies?  (Ie has anything been refactored here?)

Reply:
Moby still imports its parts using the name `github.com/docker/docker`, so nothing is changed by this rename. Actually I've checked moby/moby's dependencies, not Docker v1.11.2, but in 1.11.2 this file looks the same as in master.